treeherder/webapp/api/push.py

```diff
@@ -28,6 +28,11 @@
 logger = logging.getLogger(__name__)
 
 
+def count_unique_test_failures(failures):
```

Comment:
This is correct. But since it is a utility function that is used at 3 places now (and may get used further), I think it would be nice to have a small unit test for it.